addon/mixins/route.js

```diff
@@ -164,31 +164,20 @@ export default Ember.Mixin.create({
     this.set('_modelPath', modelPath);
     this.set('_extraParams', options);
 
-    var requestPayloadBase = {};
-    requestPayloadBase[this.get('perPageParam')] = perPage;
-    requestPayloadBase[this.get('pageParam')] = startingPage;
-
     if (typeof boundParams === 'object') {
       this.set('_boundParams', boundParams);
-      options = this._includeBoundParams(options, boundParams);
     }
 
-    var params = Ember.merge(requestPayloadBase, options);
-    let promise = this.store[this._storeFindMethod](modelName, params);
+    let promise = this._requestNextPage();
 
     promise.then(
       infinityModel => {
-        var totalPages = infinityModel.get(this.get('totalPagesParam'));
+        let totalPages = infinityModel.get(this.get('totalPagesParam'));
         this.set('currentPage', startingPage);
         this.set('_totalPages', totalPages);
         infinityModel.set('reachedInfinity', !this.get('_canLoadMore'));
```

Comment:
Feel like we could use `_finishLoad` here instead of setting this inline, right?